Sources/ScipioKit/Runner.swift

```diff
@@ -275,7 +288,9 @@ extension Runner.Options.BuildOptions {
             platforms: fetch(\.platforms, by: \.platforms),
             isSimulatorSupported: fetch(\.isSimulatorSupported, by: \.isSimulatorSupported),
             isDebugSymbolsEmbedded: fetch(\.isDebugSymbolsEmbedded, by: \.isDebugSymbolsEmbedded),
-            frameworkType: fetch(\.frameworkType, by: \.frameworkType)
+            frameworkType: fetch(\.frameworkType, by: \.frameworkType),
+            extraFlags: overridingOptions.extraFlags ?? extraFlags,
+            extraBuildParameters: overridingOptions.extraBuildParameters ?? extraBuildParameters
```

Comment:
It looks all `extraBuildParameters` will be ignored if we set `overridingOptions.extraBuildParameters`, is it okay? We don't combine them, right?

Reply:
Good point!

Let's combine two settings